xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.36.xml

```diff
@@ -25,15 +25,15 @@
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
                   <msItem class="#funerary #label">
```

Comment:
Update control vocab?

Reply:
I'm sorry I don't think I know what you mean by control vocab?

Reply:
I think having both is fine; you just want to make sure that it's `#funerary.epitaph`, and not just `#funerary` if you think it's an epitaph!